go/bundle/cmd/gen-bundle/main.go

```diff
@@ -6,11 +6,23 @@ import (
 	"log"
 	"net/url"
 	"os"
+	"strings"
 
 	"github.com/WICG/webpackage/go/bundle"
 	"github.com/WICG/webpackage/go/bundle/version"
 )
 
+type headerArgs []string
+
+func (h *headerArgs) String() string {
+	return fmt.Sprintf("%v", *h)
+}
```

Comment:
I'd write

```go
func (h headerArgs) String() string {
	return fmt.Sprintf("%v", h)
}
```

Reply:
Let me keep using `*headerArgs` here for consistency with `Set()`.

Also, `fmt.Sprintf("%v", h)` would recursively call `h.String()` so we'll need a type conversion to prevent that, which is a bit unfortunate.